README.adoc

```diff
@@ -198,10 +201,9 @@ properties is correctly handled as an error.
 To force these test cases to run in a particular order, put them in a `testSuite()` method and label
 it with a `@Test` annotation so that it automatically runs when your test class run.
 
-Finally, notice that you're using a few helper methods throughout the tests.
-You can find complete Javadocs for these in the following `finish` directory for reference:
-`finish/src/test/java/it/io/openliberty/guides/inventory/EndpointTest.java`.
-
+Finally, a `system` test class in the `src/test/java/it/io/openliberty/guides/system/SystemEndpointTest.java` file,
+has been included for you to test the basic functionality of the `inventory` service.
```

Comment:
Should be "... of the `system` service."

Reply:
Changed. Thanks.